src/lib.rs

```diff
@@ -15,5 +15,8 @@ pub mod ui;
 mod usb;
 mod vec_map;
 
+#[cfg(test)]
```

Comment:
I thought this was rather a nice use of `#[cfg(test)]` from both a convenience and semantic point of view.

Reply:
It was also essential, or you get warnings on a regular build about the code being unused.